client/gutenberg/editor/main.jsx

```diff
@@ -47,6 +52,7 @@ class GutenbergEditor extends Component {
 					hasFixedToolbar={ true }
 					post={ post }
 					onError={ noop }
+					overridePost={ { title: '' } }
```

Comment:
this is overriding the title for posts with a valid title:

| Before | After |
| - | - |
| ![screen shot 2018-09-20 at 13 41 24](https://user-images.githubusercontent.com/233601/45833450-0dc9f880-bcdb-11e8-90aa-c5406efedc7b.png) | ![screen shot 2018-09-20 at 13 41 54](https://user-images.githubusercontent.com/233601/45833474-17ebf700-bcdb-11e8-913a-e465cd2a9639.png) |

Maybe we should override the post content only when the `postType` is `auto-draft`.

Reply:
:D good catch ✨

Reply:
Well spotted! Fixed in f226d94